### PR TITLE
revert releases

### PR DESCRIPTION
**Pull request by ulugbekna**
- Revert "chore: prepare 1.15.0"
- Revert "upgrade to merlin with support for ocaml 5, upgrade olsp opam file to ocaml 5"
- Revert "Add missing declaration for functions"

### CODE REVIEW

File: submodules/lev/lev/src/lev_stubs.c [hunk 1]
```diff
@@ -57,9 +57,6 @@
     CAMLreturn(Val_long(__value));                                             \
   }
 
-CAMLextern int caml_convert_signal_number(int);
-CAMLextern int caml_rev_convert_signal_number(int);
```
Comment (rgrinberg) [inline submodules/lev/lev/src/lev_stubs.c:61]: This commit can stay. We just need it in lev as well.

Comment (ulugbekna) [reply submodules/lev/lev/src/lev_stubs.c]: remove the revert of that particular commit; here's the PR to lev: https://github.com/rgrinberg/lev/pull/86

File: submodules/lev/.github/workflows/workflow.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@ jobs:
           - ubuntu-latest
           - windows-latest
         ocaml-compiler:
-          - 5.0.x
+          - 4.14.x
```
Comment (rgrinberg) [inline submodules/lev/.github/workflows/workflow.yml:18]: I'm surprised about this change. We should be testing on 4.14 and 5.0. Could you add an appropriate change in lev for this as well?

Comment (ulugbekna) [reply submodules/lev/.github/workflows/workflow.yml:18]: https://github.com/rgrinberg/lev/pull/87

I expect something like this?

**Review by rgrinberg — APPROVED** [review]

**Comment by ulugbekna** [comment]
should I squash merge?

**Comment by rgrinberg** [comment]
Individual commits are fine here

**Pull request merged**